pretty_midi/pretty_midi.py

```diff
@@ -869,45 +961,45 @@ def get_piano_roll(self, fs=100, times=None):
             # Should interpolate
             piano_roll[note.pitch,
                        int(note.start*fs):int(note.end*fs)] += note.velocity
-
-        # Process pitch changes
-        # Need to sort the pitch bend list for the following to work
-        ordered_bends = sorted(self.pitch_bends, key=lambda bend: bend.time)
-        # Add in a bend of 0 at the end of time
-        end_bend = PitchBend(0, end_time)
-        for start_bend, end_bend in zip(ordered_bends,
-                                        ordered_bends[1:] + [end_bend]):
-            # Piano roll is already generated with everything bend = 0
-            if np.abs(start_bend.pitch) < 1:
-                continue
-            # Get integer and decimal part of bend amount
-            start_pitch = pitch_bend_to_semitones(start_bend.pitch)
-            bend_int = int(np.sign(start_pitch)*np.floor(np.abs(start_pitch)))
-            bend_decimal = np.abs(start_pitch - bend_int)
-            # Column indices effected by the bend
-            bend_range = np.r_[int(start_bend.time*fs):int(end_bend.time*fs)]
-            # Construct the bent part of the piano roll
-            bent_roll = np.zeros(piano_roll[:, bend_range].shape)
-            # Easiest to process differently depending on bend sign
-            if start_bend.pitch >= 0:
-                # First, pitch shift by the int amount
-                if bend_int is not 0:
-                    bent_roll[bend_int:] = piano_roll[:-bend_int, bend_range]
+        if include_pitch_bends:
+            # Process pitch changes
+            # Need to sort the pitch bend list for the following to work
+            ordered_bends = sorted(self.pitch_bends, key=lambda bend: bend.time)
```

Comment:
There are a bunch of lines in here which are now > 79 chars - maybe your PEP-8 checker isn't picking up on this.